variables.tf

```diff
@@ -26,6 +26,7 @@ variable "project_id" {
 variable "bucket_name" {
   description = "The name of the bucket to create"
   type        = string
+  default     = "genai-doc-summary-webhook-1234"
```

Comment:
Is this really the bucket name?

Reply:
Just the default one. It reduces one more input requirement in SiC. Do you have suggestion for a better default? While testing we use some random suffix: https://github.com/GoogleCloudPlatform/terraform-genai-doc-summarization/blob/main/examples/simple_example/main.tf#L25.